### PR TITLE
Pls review

### PR DESCRIPTION
**Pull request by putera-anami**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # belajar-kolaborasi
-commit C
+Code Review test
```
Comment (RedCyclone28) [inline README.md:2]: new line with <br> pls

**Review by RedCyclone28 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (putera-anami) [reply README.md]: done

Comment (RedCyclone28) [reply README.md]: ok

**Review by RedCyclone28 — APPROVED** [review]
approved

**Pull request merged**